src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHost.cs

```diff
@@ -68,7 +67,7 @@ public sealed class WebAssemblyHost : IAsyncDisposable
         /// </summary>
         public IServiceProvider Services => _scope.ServiceProvider;
 
-        internal WebAssemblyCultureProvider CultureProvider { get; set; } = WebAssemblyCultureProvider.Instance!;
+        internal WebAssemblyCultureProvider CultureProvider => WebAssemblyCultureProvider.Instance!;
```

Comment:
We only seem to use this in one place (within run async core).

How about we remove the property and just do
```csharp
var cultureProvider = WebAssemblyCultureProvider.Instance;
```
in the method where it is used?

Reply:
Just realized this won't be possible as we need to keep the property (setter) for use with testing (https://github.com/dotnet/aspnetcore/pull/31769/checks?check_run_id=2336355144)

Reply:
I believe https://github.com/dotnet/aspnetcore/pull/31769/commits/dd72fbc6811ae1cb57293b3b9128b3e4a599a90e should resolve this.